airbyte-integrations/connectors/source-recruitee/source_recruitee/recruitee.yaml

```diff
@@ -0,0 +1,56 @@
+version: "0.1.0"
+
+definitions:
+  schema_loader:
+    type: JsonSchema
+    file_path: "./source_sentry/schemas/{{ options.name }}.json"
+  selector:
+    extractor:
+      field_pointer: 
+        - "{{ options['name'] }}"
+  requester:
+    url_base: "https://api.recruitee.com/c/{{ config.company_id }}"
+    http_method: "GET"
+    authenticator:
+      type: BearerAuthenticator
+      api_token: "{{ config['api_key'] }}"
+  retriever:
+    record_selector:
+      $ref: "*ref(definitions.selector)"
+    paginator:
+      type: NoPagination
```

Comment:
@delenamalan the paginator it is missing.

Reply:
The offers and departments endpoints don't have any pagination mechanisms available. The candidates one does, but if you don't pass a limit or offset to it, all candidates will get returned.

Reply:
I think pagination is more important than the incremental because no pagination will lead to bad data to the sync.

Reply:
What do you mean with bad data? The endpoints return all of the data in a single response.